android/app/src/main/java/com/zbaymobile/NodeJSService.kt

```diff
@@ -63,12 +60,13 @@ class NodeJSService: Service() {
             hiddenServiceData = hiddenServiceData
         )
 
+        Thread.sleep(2000)
```

Comment:
lets always add a comment stating what we are waiting for here;
how solid is it?
perhaps a task for making it properly (waiting for a specific condtiotn?)